service/src/apps/area_name/area_name.go

```diff
@@ -0,0 +1,96 @@
+package area_name
+
+import (
+	"fmt"
+	"generic_apis/apps/utils"
+	"generic_apis/db"
+	"generic_apis/insight"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/microsoft/ApplicationInsights-Go/appinsights"
+)
+
+type handler struct {
+	db          *db.Config
+	traceparent string
+}
+
+func (conf *handler) fromDatabase(areaType, areaName string) (map[string]interface{}, error) {
+
+	var (
+		ok     bool
+		params []interface{}
+		query  string
+	)
+
+	areaType = strings.ToLower(areaType)
+
+	if areaType, ok = utils.AreaTypes[areaType]; !ok {
+		return nil, fmt.Errorf("invalid area type")
+	}
+
+	params = []interface{}{areaType, areaName}
```

Comment:
Givem that we're using `ILIKE` in the query, it might be better to use RegEx `ReplaceAll` to replace special chars + space in _area name_ with `%`. 

So let's compile the pattern `['\s,.()]` and replace all occurrences with `%` before adding the param.